server/admin/http/handler.go

```diff
@@ -394,3 +395,42 @@ func decodePeerID(id string, w http.ResponseWriter) (peer.ID, bool) {
 	}
 	return peerID, true
 }
+
+var healthCheckMH multihash.Multihash
+var healthCheckValue indexer.Value
+
+func init() {
+	provider, err := peer.Decode("12D3KooWBUNzpAz1Jfvnaag1nHBi6gbG5Q1BQm8kCfcpmt7bGKa6")
```

Comment:
We should exclude this provider+context ID from the find requests to avoid leaking bogus responses to the end user.